src/week1.org

```diff
@@ -0,0 +1,124 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I? - Kristaps
```

Comment:
This is what that slide looks like at 1920x1080 - same problem (in Chrome, at least.)

![image](https://user-images.githubusercontent.com/10482109/27838896-24ec6820-60b3-11e7-983c-58f1ac01466a.png)

I tried adjusting that margin you mentioned and it doesn't fix the issue - it just makes everything bigger but it's still the same relative amount cut off. (0.22: https://puu.sh/wATiH.png; 0: https://puu.sh/wATkL.png)

I floated the idea of background pics in Slack - I think your slide looks really good - but haven't heard back from anyone else yet.